com.sap.cloud.lm.sl.cf.web/src/main/webapp/WEB-INF/security-context.xml

```diff
@@ -35,63 +35,16 @@
 
     <sec:http-firewall ref="customHttpFirewall" />
 
-    <bean id="delegatingAuthenticationEntryPoint"
```

Comment:
Overall, I am mostly fine with the Java Based beans. Here something should be taken in mind when doing such changes - does the Java bean looks better than the xml and does it more easily express the intentions behind the Beans?